x-pack/test/security_solution_cypress/es_archives/unmapped_fields/mappings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "index",
+  "value": {
+    "aliases": {
+      "mydata-7.12.0": {
+        "is_write_index": true
+      }
+    },
+    "index": "unmapped-7.12.0-2021.03.10-000001",
+    "mappings": {
+      "dynamic": false,
```

Comment:
You should be able to get away with `dynamic: "strict"` here and still remain unmapped since below you have `index: false`, that would make anyone maintaining this better as they wouldn't accidentally do a typ-o or something and the test still passes.